packages/app-folders/src/contexts/links.tsx

```diff
@@ -40,10 +40,19 @@ interface Props {
     children: ReactNode;
 }
 
+const defaultLoading = {
+    INIT: true,
+    LIST: false,
+    LIST_MORE: false,
+    CREATE: false,
+    UPDATE: false,
+    DELETE: false
+};
+
```

Comment:
I think you should type this using the `LoadingActions` type, so that the keys are type-safe.